web/src/pages/dashboard.tsx

```diff
@@ -89,7 +89,7 @@ const DashboardPage = (props: Props): ReactElement => {
   });
 
   const CertificationsAlert = useQueryControlledAlert({
-    queryKey: QUERIES.fromTaxRegistration,
+    queryKey: QUERIES.fromForming,
```

Comment:
pebble - this change creates an entirely different issue. A user no longer sees the snack bar when they complete the formation task. You could view this by navigating to the formation task, and then clicking on a checkbox at the top to manually change the task to complete it. When the user is routed back the snackbar does not render

Reply:
Separately, I'd expect the snack bar to render when a user forms via the API, and they successfully processed the payment. I think when they return, they are also routed to the dashboard. I might be a mistaken here, I think it's worth looking into. If they are routed to the dashboard, then I expect to see the Snack bar here as well

Reply:
re: snackbar issue - brought to product's attention with an ask to resolve in another story
re: API - User is not re-routed to dashboard once formation is complete/successful.